composer.json

```diff
@@ -26,12 +25,22 @@
         "monolog/monolog": "^1.13.1",
         "smalot/pdfparser": "~0.9.24",
         "wallabag/tcpdf": "^6.2",
-        "true/punycode": "~2.0"
+        "true/punycode": "~2.0",
+        "psr/http-message": "^1.0",
+        "php-http/client-implementation": "^1.0",
+        "php-http/httplug": "^1.0",
+        "php-http/message-factory": "^1.0",
+        "php-http/discovery": "^1.0",
```

Comment:
This is fine to me.
Jumping to httplug will force a new major release (ie 2.0.0) so we can increase the minimum version of PHP in that case.